udata/settings.py

```diff
@@ -283,6 +283,30 @@ class Defaults(object):
         'rdf', 'ttl', 'n3',
     ]
 
+    ALLOWED_RESOURCES_MIMES = [
```

Comment:
I made a small visualisation on actual mime types used. We can talk about it later, but some of the most used ones are missing from this list.